.github/workflows/release.yaml

```
@@ -351,3 +362,20 @@ jobs:
               tmp/"${channel}"/* \
               '${{ env.itch_page }}':"${channel#package-}"
           done
+
+  # Upload to GitHub Pages.
+  deploy-to-pages:
+    runs-on: ubuntu-latest
+    needs: build
+    permissions:
+      # Required to upload to GitHub Pages.
```

```suggestion
  # Deploy to GitHub Pages.
  deploy-to-github-pages:
    runs-on: ubuntu-latest
    needs: build
    permissions:
      # Required to deploy to GitHub Pages.
```
IIUC now, I believe this job only deploys what we already uploaded earlier during the web build, so "deploy" was the correct verb here.